satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -98,6 +147,36 @@ func rootMutation(log *zap.Logger, service *console.Service, mailService *mailse
 						return nil, err
 					}
 
+					var referrerID *uuid.UUID
+					if len(referrerInput) > 1 {
```

Comment:
I think it would be better for readability if you did `if referrerInput != "" { ... }`